hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsStatistic.java

```diff
@@ -73,7 +73,35 @@
   READ_THROTTLES("read_throttles",
       "Total number of times a read operation is throttled."),
   WRITE_THROTTLES("write_throttles",
-      "Total number of times a write operation is throttled.");
+      "Total number of times a write operation is throttled."),
+
+  //OutputStream statistics.
```

Comment:
I've been trying to create standard names in hadoop-common; are there ones you can use. If not lets' create new ones

Reply:
Mostly OutputStream standard names in common were around BlockOutputStream since S3A uses that. So, I created these ones. Would be good to have some of these in there.